package.json

```diff
@@ -6,14 +6,17 @@
   "author": "monisahmad <monis.ahmad42@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start": "nodemon src/server.js"
+    "start": "nodemon src/server.js",
```

Comment:
Don't use development tools like `nodemon` with `start` because services rely on the `start` command to start the server. Use something like `start:dev`